client/my-sites/plans/controller.jsx

```diff
@@ -87,6 +87,10 @@ export default {
 			basePath = route.sectionify( context.path );
 		let baseAnalyticsPath;
 
+		if ( config.isEnabled( 'manage/plan-features' ) ) {
```

Comment:
I am trying to replay the flow in my mind, but it proves challenging, so I'll just drop the logic here:

Around April we tried to introduce feature pages, and highlighting of the feature we were nudging people towards. Since then we abandoned this efforts, but the highlighting is still there.

The route `/plans/features/:feature/:domain` was already there
https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plans/index.js#L87

Its handled by plansController features
https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plans/controller.jsx#L130

Which redirects it to `/plans/compare/`
Which is handled by `plansController.plansCompare `

Which you redirect to....
`/plans/features`

We are not getting a circular redirect here, because you don't redirect to a specific feature, but we're pretty close :)
Should we worry about this more? @gwwar ?